loopchain/channel/channel_service.py

```diff
@@ -181,6 +183,8 @@ def _manual_rollback(self) -> str:
 
     async def start_lft(self):
         self.__event_system = EventSystem()
+        self.__event_system.set_mediator(DelayedEventMediator)
+        self.__inner_service._task._event_system = self.__event_system  # --;.. ChannelInnerTask
```

Comment:
What does the comment,`--;.. ChannelInnerTask`, mean ?

Reply:
**Accidentally inserted comment.**
I found that `EventSystem` in `ChannelInnerService` should assigned AT THIS MOMENT to pass `IF` phrase in somewhere. So I write temporary comment to fix it.

I will remove that soon.

Reply:
Replaced to a more meaningful comment.